src/patternfly/components/Card/examples/Card.md

```diff
@@ -63,6 +63,51 @@ import './Card.css'
 {{/card}}
 ```
 
+### With subititle
+```hbs isBeta
+{{#> card card--id="card-subtitle"}}
+  {{#> card-header}}
+    {{#> card-actions}}
+      {{> card--menu-toggle}}
+      {{> card--check}}
+    {{/card-actions}}
+    {{#> card-header-main}}
+      {{> card-title card-title-text--value="This is a really really really really really really really really really really long title" card-title-text--id=(concat card--id '-check-label')}}
+    {{/card-header-main}}
+  {{/card-header}}
+  {{#> card--subtitle}} This is the card subtitle. {{/card--subtitle}}
```

Comment:
Sorry about this but I forgot to mention that this subtitle should go inside of the card title, after the card title text. So the structure should be:

```
card__title
  card__title-text
  card__subtitle
```

And it only needs 3 styles - padding-block-start, font-size, color.